plugin/pkg/admission/securitycontext/scdeny/admission.go

```diff
@@ -23,17 +23,26 @@ import (
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apiserver/pkg/admission"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/klog/v2"
 	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/features"
 )
 
 // PluginName indicates name of admission plugin.
 const PluginName = "SecurityContextDeny"
 
 // Register registers a plugin
 func Register(plugins *admission.Plugins) {
-	plugins.Register(PluginName, func(config io.Reader) (admission.Interface, error) {
```

Comment:
check the feature gate inside the constructor and error with a message about this being an alpha feature, requiring the SecurityContextDeny feature gate to be enabled, and that the feature is planned to be removed:

```go
plugins.Register(PluginName, func(config io.Reader) (admission.Interface, error) {
  if utilfeature.DefaultFeatureGate.Enabled(features.SecurityContextDeny) {
    return nil, fmt.Errorf("...")
  }
  return NewSecurityContextDeny(), nil
})
```